gradle.properties

```diff
@@ -17,4 +17,4 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-version=1.0.2-SNAPSHOT
+version=1.0.3
```

Comment:
The way we usually manage versions is only to commit SNAPSHOT to git, and when we release a version, we shift the version, release, and then bump to the next SNAPSHOT. So we never see a non-SNAPSHOT committed.

Reply:
I put the `SNAPSHOT` back in.